mvc-login/src/main/java/com/auth0/example/SecurityConfig.java

```diff
@@ -1,41 +1,45 @@
 package com.auth0.example;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.web.SecurityFilterChain;
-import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
+import org.springframework.security.web.authentication.logout.LogoutHandler;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 
-@EnableWebSecurity
-public class SecurityConfig {
+import java.io.IOException;
 
-    private final LogoutHandler logoutHandler;
+import static org.springframework.security.config.Customizer.withDefaults;
 
-    public SecurityConfig(LogoutHandler logoutHandler) {
-        this.logoutHandler = logoutHandler;
-    }
+@Configuration
+@EnableMethodSecurity(securedEnabled = true)
```

Comment:
Is this necessary? I don't see `@Secured` being used.

Reply:
You are right, it is not necessary; new commit pushed to remove it.